featuretools/tests/primitive_tests/test_feature_serialization.py

```diff
@@ -17,8 +17,8 @@
 BUCKET_NAME = "test-bucket"
 WRITE_KEY_NAME = "test-key"
 TEST_S3_URL = "s3://{}/{}".format(BUCKET_NAME, WRITE_KEY_NAME)
-S3_URL = "s3://featuretools-static/test_feature_serialization_1.0.0"
-URL = "https://featuretools-static.s3.amazonaws.com/test_feature_serialization_1.0.0"
+S3_URL = "s3://featuretools-static/test_feature_serialization_feature_schema_3.2.0_entityset_schema_2.0.0.json"
+URL = "https://featuretools-static.s3.amazonaws.com/test_feature_serialization_feature_schema_3.2.0_entityset_schema_2.0.0.json"
```

Comment:
these urls are pretty long, maybe we should move the file name to a separate variable and construct the urls using the variable